community/lld/APKBUILD

```diff
@@ -33,6 +33,7 @@ build() {
 		-DCMAKE_INSTALL_PREFIX=/usr \
 		\
 		-DCMAKE_SKIP_INSTALL_RPATH=ON \
+		-DLLVM_EXTERNAL_LIT=/usr/lib/llvm7/bin/lit \
```

Comment:
Can we also use somthing like  `-DLLVM_EXTERNAL_LIT=/usr/lib/llvm${pkgver%%.*}/bin/lit` here?

Reply:
Tests are not passing as expected. Any hints?